src/main/java/org/apache/maven/plugins/scmpublish/AbstractScmPublishMojo.java

```diff
@@ -107,13 +110,13 @@ public abstract class AbstractScmPublishMojo extends AbstractMojo {
     protected String subDirectory;
 
     /**
-     * Display list of added, deleted, and changed files, but do not do any actual SCM operations.
+     * If set to {@true} displays list of added, deleted, and changed files, but does not do any actual SCM operations.
```

Comment:
Perhaps `{@code true}`?

Reply:
Indeed, fixed now in https://github.com/apache/maven-scm-publish-plugin/commit/ecf4fbfa2a88f673c5b9a86b1171bf2d6972278a.